### DISCUSSION TITLE
WinMerge & Windows 'properties' differ

### DISCUSSION THREAD

**Discussion by MRK6310** [WinMerge/winmerge#1736] [Q&A]
I'd like to ensure two USB memory sticks have identical contents. 
Each memory stick is an 8GB SanDisk 'Cruzer Blade'.
WinMerge says there's no difference.
Windows File Explorer says one stick has one more folder, and two more files than  the other.  
Could anyone suggest why the Winmerge and Windows results are different?

**Comment by sdottaka** [comment]
Does checking the View→Show Skipped Items menu item change the number of items displayed?

If so, it may be filtered out by file filters or due to a bug introduced in WinMerge version 2.16.26.

If you are using version 2.16.26, please upgrade to 2.16.28.
Also, the currently used file filter is displayed in the bottom right corner as shown in the attached image, so please tell me what is displayed.

![image](https://user-images.githubusercontent.com/98126/221866599-34344b49-9166-4e60-8e33-e5a8cb2dd803.png)

**Comment by MRK6310** [comment]
Thanks for your help.
I'm using WinMerge 2.16.14 on a 32-bit laptop PC running under Windows 7.Is WinMerge 2.16.28 compatible with this setup?
Checking the "Show skipped items" option in the WinMerge View options produces no change in the number of items shown.
I tried different file filters. The current filter displayed is "Full Contents".
I attach two screenshots. One shows the WinMerge comparison, the other shows the Windows Properties comparison. 

I hope this information can help in finding out why the comparison results differ between WinMerge and Windows Properties.

-----Original Message-----
From: Takashi Sawanaka ***@***.***>
To: WinMerge/winmerge ***@***.***>
CC: MRK6310 ***@***.***>; Author ***@***.***>
Sent: Tue, 28 Feb 2023 13:29
Subject: Re: [WinMerge/winmerge] WinMerge & Windows 'properties' differ (Discussion #1736)

Does checking the View→Show Skipped Items menu item change the number of items displayed?If so, it may be filtered out by file filters or due to a bug introduced in WinMerge version 2.16.26.If you are using version 2.16.26, please upgrade to 2.16.28.
Also, the currently used file filter is displayed in the bottom right corner as shown in the attached image, so please tell me what is displayed.—

**Comment by sdottaka** [comment]
WinMerge 2.16.28 should also work on Windows 7 32-bit version.

If the number of items does not change in Show Skipped Items, it is not likely due to the file filter.

Are all menu items checked except for Show Skipped Items?

Unfortunately, the attached image is not visible at the following URL. Could you please attach the file to the following URL?

https://github.com/WinMerge/winmerge/discussions/1736

**Reply by MRK6310** [reply]
All Items are checked, including Skipped Items.

I'm sorry you couldn't see theattachments. Hopefully they'll be sent along with this reply....

![WinMerge Screenshot](https://user-images.githubusercontent.com/126577378/221902324-80dba3cc-9bff-4a1d-89d9-6c33ff229b67.jpg)
![Windows Proerties Compared](https://user-images.githubusercontent.com/126577378/221902374-9537f13f-226b-4b8f-bdea-392bd8753e0b.jpg)

**Comment by sdottaka** [comment]
Another possibility is that a folder has hidden attributes and is not visible in Explorer.

WinMerge does not compare the attributes of files and folders.

Try displaying the Left/Right Attributes column as follows. If it contains an "H", it is a hidden file or folder.

![image](https://user-images.githubusercontent.com/98126/222005411-3bd1d016-a233-488d-8518-a8f52899af43.png)
